### PR TITLE
Typos pointed out in issue 1021

### LINKED ISSUE #1021: Typos in v.1.1
This is KAMITSUNA Shuji from Japan.

I translated the v.1.1 spec into Japanese.
Thank you for the great spec.
http://www.asahi-net.or.jp/~ax2s-kmtn/internet/did/REC-vc-data-model-20220303.html

I found some possible typos:
(1) in the image, Figure 10: "Repeateable" might be "Repeatable".
(2) in the alt attribute, Figure 12: "ia" might be "via".
(3) in the alt attribute, Figure 13: "e.g., Parent, Pet Owner, Travel Agent" might be "e.g., Pet Owner, Travel Agent". That is, no mention about parent in the image.

It seems like some of these remain in v.2.0.

Additionally, according to the W3C Link Cheker, there are issues with the URLs.
https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2022%2FREC-vc-data-model-20220303%2F&hide_type=all&depth=&check=Check

Best regards,

### PR DESCRIPTION
**Pull request by bumblefudge**
- Fixed 1, 2 and the links in CURRENT draft
- Also updated @David-Chadwick 's affiliation while I was there
- 3 was only present in drafts

Should I also fix these in the previous drafts?  If so do I have to update an errata list somewhere, or is that unnecessary for such minor typos?

Closes #1021

***
<a href="https://pr-preview.s3.amazonaws.com/bumblefudge/vc-data-model/pull/1022.html" title="Last updated on Feb 1, 2023, 9:37 AM UTC (027bf72)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1022/c344fa0...bumblefudge:027bf72.html" title="Last updated on Feb 1, 2023, 9:37 AM UTC (027bf72)">Diff</a>

### CODE REVIEW

File: diagrams/ecosystemdetail.svg [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 <path class="link" d="m430 175c100 -100 120 -100 125 10"/>
 <text x="550" y="120" role="heading" aria-level="2">Transfer</text>
-<text x="560" y="135">Repeateable</text>
+<text x="560" y="135">Repeatable</text>
```
Comment (TallTed) [inline diagrams/ecosystemdetail.svg:53]: to match the other lowercase instances of `repeatable` in the diagram
```suggestion
<text x="560" y="135">repeatable</text>
```

Comment (bumblefudge) [reply diagrams/ecosystemdetail.svg:53]: 🤦‍♀️

**Review by msporny — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-02-01-vcwg#section3-4) on 2023-02-01

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.4. Typos pointed out in issue 1021 (pr vc-data-model#1022)

_See github pull request [vc-data-model#1022](https://github.com/w3c/vc-data-model/pull/1022)._




**Manu Sporny:** PR 1022 this one was put in by Juan which fixes some editorial issues that were just wrong. Also updated an image. This one looks really good, but would be good to have more reviewers..  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**